src/Breadcrumbs/Breadcrumb.php

```diff
@@ -142,27 +137,23 @@ public function withType($type)
     /**
      * Gets the breadcrumb level.
      *
-     * @return string
+     * @return Severity
      */
-    public function getLevel()
+    public function getLevel(): Severity
```

Comment:
Maybe we can rename it to `getSeverity()` and rename the property too, for consistency?

Reply:
It was named this way because the SDK docs refer to the `level` attribute. Another inconsistency that should be fixed @HazAT?

Reply:
Actually, the levels of a breadcrumb are not 1:1 to event Severity.
https://docs.sentry.io/clientdev/interfaces/breadcrumbs/
Maybe we shouldn't use Severity for breadcrumbs.

Reply:
Oh, didn't notice that. I will then revert to the change to the constants